csharp/extractor/Semmle.Extraction.CSharp/Entities/CommentLine.cs

```diff
@@ -1,25 +1,25 @@
-using Semmle.Extraction.CommentProcessing;
 using Semmle.Extraction.Entities;
 using System.IO;
+using System;
 
 namespace Semmle.Extraction.CSharp.Entities
 {
-    internal class CommentLine : CachedEntity<(Microsoft.CodeAnalysis.Location, string)>, ICommentLine
+    internal class CommentLine : CachedEntity<Tuple<Microsoft.CodeAnalysis.Location, string>>
```

Comment:
There's a non-cosmetic change here: `ValueTuple -> Tuple`. This way the `class` generic type constraint could be applied on the type parameter of the base class.

Reply:
I don't think this change is needed if you replace the `class` constraint with `notnull`.